contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -14,12 +14,31 @@ import { XVSRewardsHelper } from "./XVSRewardsHelper.sol";
  * @notice This facet contract contains all the external pre-hook functions related to vToken
  */
 contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
+    /// @notice The address of the Keyring contract.
+    /// @dev This could have been immutable, but not possible in upgradeable contracts.
+    /// @dev Consumes one storage slot.
+    IKeyringChecker public keyringChecker;
+
+    /// @notice The policyId to use with the Keyring contract.
+    /// @dev This could have been immutable, but not possible in upgradeable contracts.
+    /// @dev Consumes one storage slot.
+    uint32 public keyringPolicyId;
+
+    /// @notice Whether the keyring guard is enabled.
+    bool public keyringGuardEnabled;
+
```

Comment:
Facets cannot have attributes. They should be defined in the file `contracts/Comptroller/ComptrollerStorage.sol`, inside a new versioned contract (`ComptrollerV17Storage is ComptrollerV16Storage`). `Diamond` and `FacetBase` contracts should be updated to extend the new storage